public/index.html

```diff
@@ -14,6 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <script src="https://www.gstatic.com/firebasejs/8.3.2/firebase-app.js"></script>
```

Comment:
we will delete files src/firebase/index.js and src/firebase.js to avoid confustion, redundancy and incorrect references.  Your changes will be consolidated into one branch.

Reply:
Sounds good, feel free to make the changes in this branch for simplicity. Creating branches off of branches can lead to complicated `git` situations, merge conflicts and similar.

Removing this `<script>` reference will remove this error:
![image](https://user-images.githubusercontent.com/5881537/114279286-ac78ea80-99e8-11eb-86da-46b84c8047b1.png)

Reply:
Nice catch! complete.